SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/TestLayeredLBStrategy.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/DefaultPawn.h"
+
+#include "LoadBalancing/GridBasedLBStrategy.h"
```

Comment:
nit, can u move this above engine includes

Reply:
done